src/components/xDragBoxLayer.ts

```diff
@@ -17,7 +17,7 @@ export module Components {
 
     public computeLayout(origin?: Point, availableWidth?: number, availableHeight?: number) {
       super.computeLayout(origin, availableWidth, availableHeight);
-      this.bounds(this.bounds()); // set correct bounds when width/height changes
+      this._setBoundsWithoutChangingMode(this.bounds()); // set correct bounds when width/height changes
```

Comment:
Whatever is the fix, can we not have end of the line comments? Instead move them on the line above.